tests/ui_json_test.py

```diff
@@ -80,6 +80,48 @@ def get_workspace(directory: str | Path):
     return workspace
 
 
+def get_workspace_drillholes(directory: str | Path):
```

Comment:
This is more or less duplication of `from .drillhole_v4_0_test import create_drillholes`
Can you remove here and import instead?